CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -2,9 +2,14 @@
 
 Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.4.md).
 
+## v3.5.12 (tbd)
+
+## Dependencies
+- Compile binaries using [go 1.20.12](https://github.com/etcd-io/etcd/pull/17077)
```

Comment:
Compiling with go1.20.12 went into v3.5.12

Reply:
https://github.com/etcd-io/etcd/commits/v3.5.11

Reply:
Sorry, mixed things up. Should be fine now